packages/bpk-component-tooltip/src/BpkTooltipPortal-test.js

```diff
@@ -52,7 +52,7 @@ describe('BpkTooltipPortal', () => {
       <BpkTooltipPortal
         id="my-tooltip"
         target={<div>target</div>}
-        className="my-custom-class"
+        portalClassName="my-custom-class"
```

Comment:
There should be two tests for this - one for custom `className` and one for `portalClassName`.

This current one is a mixture of the two (from a test name and component prop point of view)